Makefile

```diff
@@ -2,7 +2,7 @@ CWD = $(shell pwd)
 SRC_DIR = ${CWD}/pootle
 DOCS_DIR = ${CWD}/docs
 STATIC_DIR = ${SRC_DIR}/static
-ASSETS_DIR = $(shell python -c "from pootle.settings import *; print(STATIC_ROOT)")
+ASSETS_DIR = ${SRC_DIR}/assets
```

Comment:
I rather added [a comment on STATIC_ROOT](https://github.com/evernote/zing/pull/237/commits/b739b71e2c26d351d790c093b0f6a808480bc660) since it makes more sense to me.